webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js

```diff
@@ -63,13 +65,29 @@ class RepositorySetRepository extends Component {
     };
   }
 
+  yStreamHelp = () => (
+    <span translate>
+      This repository is not suggested. Please see additional&nbsp;
+      <a href="https://access.redhat.com/articles/1586183">documentation</a>
+      &nbsp;prior to use.
+    </span>
+  );
+
+
   render() {
-    const { arch, releasever } = this.props;
+    const { arch, releasever, type } = this.props;
+    const shouldDeemphasize = () => type !== 'kickstart' && yStream(releasever);
+    const repositoryHeading = () => (
+      <span>
+        {arch} {releasever}
+        {shouldDeemphasize() ? (<FieldLevelHelp content={this.yStreamHelp()} />) : null}
+      </span>
+    );
```

Comment:
sorry to nitpick, if you are moving things out of `render()`, you can move these two as well, they may have to take arguments, but should be able to be moved just for consistency.